app/templates/direct-award/end-search.html

```diff
@@ -65,7 +65,11 @@ <h2 class="heading-xmedium">After your search has ended</h2>
         method="POST"
         >
         <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
-        <button class="button-save end-your-search-button">End search and continue</button>
+        <button class="button-save end-your-search-button"
+          {% if disable_end_search_btn %} 
+           disabled="disabled"
+          {% endif %}
```

Comment:
It would be a bit neater and more consistent with elsewhere if this was all inline, i.e.: `<button class="button-save end-your-search-button"{% if disable_end_search_btn %} disabled="disabled"{% endif %}>`